### ISSUE TITLE
Consider removing dontrun from examples

### ISSUE THREAD

**Issue by johnbradley** [phenoscape/rphenoscape#204]
Some of the function documentation examples are wrapped in `dontrun` blocks even though the code runs fine.
Consider removing these `dontrun` blocks so the HTML documentation will be more informative.

**Comment by johnbradley** [comment]
I removed all the `dontrun` lines from the examples an ran the Check command.
The following functions were flagged:
```
 Examples with CPU or elapsed time > 5s
                        user system elapsed
  pk_get_study_meta  141.037  2.526 151.177
  pk_get_study       111.967  1.598 118.854
  get_study_data      42.856  0.514  57.501
  pk_ontotrace         5.823  0.141   7.752
  get_ontotrace_data   4.329  0.093  12.029
  nexml_drop           4.129  0.033   6.875
  similarity           0.490  0.033  10.978
```
I created a branch with the other examples that were <= 5 seconds:
https://github.com/phenoscape/rphenoscape/tree/204-remove-dontruns

**Comment by johnbradley** [comment]
Fixed by #207
